DirectEmployers/MyJobs
# Use base-react.html on root url if user is authenticated [PD-3864]

## What's this PR do?
Toggles what is displayed on `/` if the user is logged in. This does not touch the login action so logging in still redirects to the profile page.
## Screenshots (if appropriate)
Unauthenticated:
![2017-10-06-083335_955x745_scrot](https://user-images.githubusercontent.com/3379265/31277937-2f0b2e06-aa71-11e7-84e4-539b5b43a607.png)

Super wonderful authenticated styles:
![2017-10-06-083419_1440x81_scrot](https://user-images.githubusercontent.com/3379265/31277956-42951ca2-aa71-11e7-9ff3-89bd29f9a5ee.png)
## Where should the reviewer start?
The `home` view, I suppose.
## How should this be manually tested?
Log out. You see a form. Log in and go back to `/`. You see my amazing CSS skills.
## Any background context you want to provide?
This is the beginning of our dashboard work. It was decided that the unauthenticated portion could remain as-is.
## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
